tests/parsing/ConfParserTest.py.orig

```diff
@@ -0,0 +1,206 @@
+import os
+import tempfile
+import unittest
+from collections import OrderedDict
+import logging
+
+from coalib.parsing.ConfParser import ConfParser
+from coalib.settings.Section import Section
+
+
+class ConfParserTest(unittest.TestCase):
+    example_file = """setting = without_section
+    [foo]
+    to be ignored
```

Comment:
Your commit message says you are only adding language definitions, then why is this file here?

Reply:
I don't know how this got generated.Will remove this at once.